web/src/narrow_banner.js

```diff
@@ -148,6 +148,13 @@ function pick_empty_narrow_banner() {
             return SPECTATOR_STREAM_NARROW_BANNER;
         }
 
+        // For empty is-operator search, we only display the title.
+        if (current_filter.operands("is").length > 0) {
+            return {
+                title: $t({defaultMessage: "There are no messages here."}),
```

Comment:
Let's make this translated string a constant (maybe `default_title`), right before `default_banner`. And then use that in both the default banner and here. Then, if that is ever updated, it only needs to be changed in one place vs two.

Reply:
Sure, working on it.

Reply:
That sounds great. I have changed the code to use only one constant.